src/types/topology/Link.types.ts

```diff
@@ -19,7 +19,7 @@ export interface TopologyAbstractLink extends TopologyComponent {
   position: () => { x1: number; x2: number; y1: number; y2: number };
   line: () => any; // TODO: line
   topology: () => any;
-  id: () => number | string;
+  id: () => NxId | NxId;
```

Comment:
Why is this OR'ed?

Reply:
I dont think this type is working correctly but the user can either do `link.id()` or `link.id = 123`. I want them to be able to call the function or set it to NxID ie string/number. Thoughts/ do you know how to do this properly?

Reply:
Ah I see. My only suggestion would be to add parentheses to make it clearer, ie `id: (() => NxId) | NxId;`